components/style/values/specified/length.rs

```diff
@@ -798,12 +784,7 @@ impl ToCss for CalcLengthOrPercentage {
             };
         }
 
-        let count = count!(ch, em, ex, absolute, rem, vh, vmax, vmin, vw, percentage);
-        assert!(count > 0);
```

Comment:
This assert was unnecessary because `CalcLengthOrPercentage` fails when it's empty.